python/lsst/sims/featureScheduler/features.py

```diff
@@ -69,6 +69,17 @@ def update_conditions(self, conditions, *kwargs):
         self.feature['az'] = az
 
 
+class BulkCloudCover(BaseConditionsFeature):
+
+    def __init__(self):
+        """Store the bulk cloud cover.
+        """
+        self.feature = dict()
+
+    def update_conditions(self, conditions, *kwargs):
+        self.feature['bulk_cloud_cover'] = conditions['clouds']
```

Comment:
My mistake here! This should actually be 

```
self.feature = conditions['clouds']
```

Reply:
Fixed.